Admin/Custom.Xperience.Stripe.Admin/Controllers/StripeController.cs

```diff
@@ -1,22 +1,24 @@
-﻿using System.Net.Http;
+﻿using System;
+using System.Linq;
+using System.Net.Http;
 using System.Net;
 using System.Web.Http;
-using CMS.Core;
-using System;
-using Stripe;
-using CMS.Helpers;
-using System.Linq;
-using Stripe.Checkout;
-using CMS.Ecommerce;
 using System.IO;
 using System.Web;
 using System.Threading.Tasks;
 using System.Collections.Generic;
+using Stripe;
+using Stripe.Checkout;
+using CMS.Helpers;
+using CMS.Core;
+using CMS.Ecommerce;
+
 
 namespace Custom.Xperience.Stripe.Endpoint
 {
     public class StripeController : ApiController
     {
+
         private IEventLogService eventLogService;
         private IAppSettingsService appSettingsService;
         public StripeController()
```

Comment:
I would inject all InfoProviders here (OrderInfoProvider, etc) in case a customer has injected their own